intranet/apps/polls/models.py

```diff
@@ -94,6 +94,34 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        groups = DjangoGroup.objects.none()
+        if self.groups.exists():
+            q_objects = Q()
+            for g in self.groups.all():
+                q_objects.add(Q(name=g), Q.OR)
+            groups = DjangoGroup.objects.filter(q_objects)
+            users = get_user_model().objects.none()
+            for g in groups:
+                users = users.union(g.user_set.all())
+            return users.count()
+        else:
+            return get_user_model().objects.count() - 1
+
+    def get_percentage_voted(self, voted, able):
+        percentage = 0
+        try:
+            percentage = voted / able
+        except ZeroDivisionError:  # there are no eligible voters
+            pass
+        return "{:.1%}".format(percentage)
+
+    def get_voted_string(self):
+        users_voted = len(self.get_users_voted())
+        users_able = self.get_num_eligible_voters()
+        percent = self.get_percentage_voted(users_voted, users_able)
+        return "{} out of {} ({}) of eligible users voted in this poll.".format(str(users_voted), str(users_able), percent)
```

Comment:
```suggestion
        return "{} out of {} ({}) eligible users voted in this poll.".format(users_voted, users_able, percent)
```

Calling `str` is unnecessary